Garhal/Garhal.vcxproj

```diff
@@ -118,6 +118,7 @@
     <ClInclude Include="garhal.h" />
     <ClInclude Include="gstructs.h" />
     <ClInclude Include="hide.h" />
+    <ClInclude Include="ioctlcodes.h" />
```

Comment:
Please place the global headers to a completely different project folder. Name It "GarhalGlobals"

Reply:
@lufog ?

Reply:
Sorry, at this moment, I don't have any time to work with this. You can merge pr and rename files as you wish or close this pr.